wled00/wled00.ino

```diff
@@ -160,7 +157,7 @@ bool syncToggleReceive = false;               //UIs which only have a single but
 
 //Sync CONFIG
 bool buttonEnabled =  true;
-bool irEnabled     = false;                   //Infrared receiver
+byte irEnabled     =  0;                      //Infrared receiver
```

Comment:
this changes the "ON/OFF" checkbox to a value entry field (0 to 4) with 0=disabled, 1=1st remote, 2=2nd remote etc. /// 4 different IR remotes in total.